pkg/mimirtool/client/client.go

```diff
@@ -119,12 +122,26 @@ func (r *MimirClient) doRequest(path, method string, payload []byte) (*http.Resp
 		return nil, err
 	}
 
+	if (r.user != "" || r.key != "") && r.authToken != "" {
```

Comment:
There are three `if` statements one next to another working on `r.user`, `r.key` and `r.authToken`. Can we rewrite them into single `switch` statement instead?

Reply:
Makes sense, will update it asap.